types/async-redis/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["es6"],
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictFunctionTypes": true,
+    "strictNullChecks": true,
+    "baseUrl": "../",
+    "typeRoots": ["../"],
+    "types": [],
+    "noEmit": true,
+    "forceConsistentCasingInFileNames": true,
+    "esModuleInterop": true
```

Comment:
This isn't needed and should be removed to make sure that the types work with old code bases that don't have it turned on.

Reply:
Which lines should I remove? from 11th to 14th or only 13th and 14th ?

Reply:
It still works without "esModuleInterop": true. But checks have failed without types, noEmit and forceConsistentCasingInFileNames. It's ok ?

Reply:
Yes; forceConsistentCasingInFileNames makes compilation stricter; esModuleInterop makes it more lenient. It's fine to have options on DT that compilation stricter.